### DISCUSSION TITLE
Changes in config.json file are ignored

### DISCUSSION THREAD

**Discussion by lalalagartija** [GMOD/jbrowse-components#3597] [General]
Hi, 
I had a bug with the display of one track so I changed it inf the config file without success. Finally I completely deleted it from the config.json file. All the changes I made where ignored (same display id was given in error message every time although I changed it for example) and now that I deleted it, the track is still available for selection. 

Is there something I don't get about how Jbrowse works ? 
Thanks

The error : 
"Error: [mobx-state-tree] Cannot resolve a reference to type 'LinearBasicDisplayConfigurationSchema' with id: 'basicdisplay' unambigously, there are multiple candidates: /jbrowse/tracks/22/displays/0, /jbrowse/tracks/23/displays/0"

**Comment by cmdcolin** [comment]
Hello. If you paste your config it may be able to help. Note that sometimes, the browser can cache the old config.json file, and clearing your local browser cache can help fix that. Ctrl+Shift+R is a way to do that quickly sometimes https://bacreative.com.au/how-to-hard-refresh-browser-and-clear-cache-using-windows-or-linux/

If you paste your config it might be able to help but if you think you already fixed it might not need to paste it

**Comment by lalalagartija** [comment]
That was indeed the problem thank you !
